dbt/schema_tester.py

```diff
@@ -0,0 +1,174 @@
+import os
+import yaml
+
+from dbt.runner import RedshiftTarget
+
+QUERY_VALIDATE_NOT_NULL = """
```

Comment:
there's a `templates.py` file that's probably a good place for this. I'll swizzle it aroudn